zha/application/platforms/light/__init__.py

```diff
@@ -333,7 +333,7 @@ async def async_turn_on(self, **kwargs: Any) -> None:
 
         t_log = {}
 
-        if new_color_provided_while_off:
+        if new_color_provided_while_off and self._level_cluster_handler is not None:
```

Comment:
Do we want to put this in `new_color_provided_while_off` variable above where the "brightness supported" is also checked?